sig/datadog/appsec/event.rbs

```diff
@@ -13,14 +13,18 @@ module Datadog
       def self.record_via_span: (Datadog::Tracing::SpanOperation, *untyped events) -> untyped
 
       def self.build_service_entry_tags: (Array[Hash[::Symbol, untyped]] event_group) -> Hash[::String, untyped]
-
+                                       
+      def self.add_tags: (Datadog::AppSec::Scope scope, Datadog::AppSec::WAF::Result waf_result) -> untyped
+                       
       private
 
       def self.compressed_and_base64_encoded: (untyped value) -> untyped
 
       def self.json_parse: (untyped value) -> untyped
 
       def self.gzip: (untyped value) -> untyped
+                   
+      def self.add_distributed_tags: (untyped trace) -> untyped
```

Comment:
May I ask to type it? (Not sure about `trace` type, but I think it could be defined too.

```suggestion
      def self.add_distributed_tags: (untyped trace) -> void
```

Reply:
Fixed in 22f18a8